packages/pwa/src/components/feedSubscriptions/FeedSubscriptionSettings.tsx

```
@@ -15,6 +15,7 @@
 import {DayOfWeek} from '@shared/types/datetime.types';
 import {DeliveryScheduleType} from '@shared/types/deliverySchedules.types';
 import type {DeliverySchedule} from '@shared/types/deliverySchedules.types';
+import {FeedSourceType} from '@shared/types/feedSources.types';
```

ESLint

[error] 'FeedSourceType' is defined but never used.

(unused-imports/no-unused-imports)